From 2d5641358cab49650bb5f02eb258687c13bda276 Mon Sep 17 00:00:00 2001
From: AnilKumar Ch <anilkumar@ti.com>
Date: Tue, 26 Aug 2014 11:13:17 +0530
Subject: [PATCH 1080/1587] ARM: dts: AM33XX: Enable system power off control
 in am335x-bone

Enable system power off control for BeagleBone in am335x-bone-common.dts
file under rtc node. RTC is the incharge of controlling the system power.
This flag is used by the driver to hook up the pm_power_off system call.

Signed-off-by: AnilKumar Ch <anilkumar@ti.com>
[j-keerthy@ti.com] ported to 3.14
Signed-off-by: Keerthy <j-keerthy@ti.com>
[zou: Original patch taken from
ti-sdk-am335x-evm-08.00.00.00-Linux-x86-Install.bin]
Signed-off-by: Cao Zou <cao.zou@windriver.com>
---
 arch/arm/boot/dts/am335x-bone-common.dtsi |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/am335x-bone-common.dtsi b/arch/arm/boot/dts/am335x-bone-common.dtsi
index 1640784..c4ae846 100644
--- a/arch/arm/boot/dts/am335x-bone-common.dtsi
+++ b/arch/arm/boot/dts/am335x-bone-common.dtsi
@@ -293,6 +293,10 @@
 		ldo4_reg: regulator@6 {
 			regulator-always-on;
 		};
+
+		rtc@44e3e000 {
+			ti,system-power-controller;
+		};
 	};
 };
 
-- 
1.7.5.4

